notebooks/en/advanced_rag.ipynb

```diff
@@ -6,7 +6,7 @@
     "id": "hUCaGdAj9-9F"
```

Comment:
<p>The addition of <code>.ipynb</code> will break the link, which for now works (test it <a href="https://huggingface.co/learn/cookbook/advanced_rag" target="_blank">here</a>).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Ah, interesting... That makes sense. I tried the notebook in VS Code. There the link w/o `.ipynb` suffix didn't work. So VSCode seems to interpret these links differently. 👍

Reply:
Done!

Reply:
@aymeric-roucher Can I do more to help with the merge? I have pushed the change as you suggested. 
Thanks and have a great day!